setup.cfg

```diff
@@ -33,6 +33,7 @@ setup_requires =
     numpy>=1.16.6,<1.20
     scipy>=1.0
     cython>=0.29.20
+    packaging
```

Comment:
You need to specify this in both `setup_requires` and `install_requires` - the former is about the build process, the latter is about what you need at installation.